Bottom: b63b7f4eb2bfe29ca259f0a42e6246f08eeadf04
Top:    f6f6919965d0a5e2557a09b588c1d4be8bb72073
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 15:40:16 +0100

Refresh of hid-gtk-drop-in-gl-renderer-wo

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 71759ca..0be44de 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1188,3 +1188,23 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
 
   return pixmap;
 }
+
+HID *
+ghid_request_debug_draw (void)
+{
+  ghid_start_drawing (gport);
+  return &ghid_hid;
+}
+
+void
+ghid_flush_debug_draw (void)
+{
+//  gdk_window_process_all_updates ();
+}
+
+void
+ghid_finish_debug_draw (void)
+{
+  /* No special tear down requirements
+   */
+}
